pkg/local_object_storage/shard/get.go

```diff
@@ -140,7 +144,15 @@ func (s *Shard) fetchObjectData(addr oid.Address, skipMeta bool, cb storFetcher,
 		return nil, true, fmt.Errorf("can't fetch blobovnicza id from metabase: %w", err)
 	}
 
-	res, err := cb(s.blobStor, mExRes.StorageID())
+	storageID := mExRes.StorageID()
```

Comment:
Shouldn't this commit be a part of the previous one?

Reply:
i think no: they fix different bugs